app/controllers/lunches/details/index.js

```diff
@@ -148,7 +148,7 @@ function showCamera() {
 	}
 	
 	// Send red sqare on Simulator
-	if (Ti.App.getDeployType() == 'development') {
+	if (utils.isEmulator()) {
```

Comment:
well "development"  I canT test on  my device now, it says im on simulator what is wrong!